src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -213,8 +214,8 @@ class PrisonerIepLevelReviewService(
   }
 
   private suspend fun getIepLevelForReviewType(prisonerInfo: PrisonerAtLocation, reviewType: ReviewType): String {
-    val iepLevelsForPrison = iepLevelService.getIepLevelsForPrison(prisonerInfo.agencyId)
-    val defaultLevelCode = iepLevelService.chooseDefaultLevel(prisonerInfo.agencyId, iepLevelsForPrison)
+    val prisonIncentiveLevels = prisonIncentiveLevelService.getActivePrisonIncentiveLevels(prisonerInfo.agencyId)
+    val defaultLevelCode = iepLevelService.chooseDefaultLevel(prisonerInfo.agencyId, prisonIncentiveLevels)
```

Comment:
We could stop using `chooseDefaultLevel()`. Wouldn't `prisonIncentiveLevels .first(PrisonIncentiveLevel::defaultOnAdmission).levelCode` be equivalent now that prisons must have a default level?